src/screens/level.py

```diff
@@ -258,10 +260,14 @@ def create_particle(self, sprite):
     def apply_tool(self, tool: FarmingTool, pos, entity):
         match tool:
             case FarmingTool.AXE:
-                for tree in self.tree_sprites:
-                    if tree.rect.collidepoint(pos):
-                        tree.hit(entity)
-                        self.sounds['axe'].play()
+                for tree in pygame.sprite.spritecollide(
```

Comment:
maybe add a transition variable for lambda function

Reply:
What transition?

Reply:
i messed up with code selection, you have a lamba func in sprite collide

Reply:
That does not answer my question of "what transition?"

Reply:
A transitional (or intermediary) variable to clarify the code